src/soundio/soundmanagerutil.h

```diff
@@ -102,27 +103,52 @@ class AudioPath {
         return qHash(path.hashValue(), seed);
     }
 
-    friend bool operator==(
-            const AudioPath& lhs,
-            const AudioPath& rhs) {
-        // Exclude m_channelGroup from comparison!
-        // See also: hashValue()/qHash()
-        // TODO: Why??
```

Comment:
no, it came from the previous code and I didn't bother much into it. likely because `m_channelGroup` is synthesized or not important for distinguishing instances of the class.